modules/functions-list-kn.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies
+
+// * /serverless/cli_reference/kn-func-ref.adoc
+
+[id="functions-list-kn_{context}"]
+= Listing existing functions
+
+You can list existing functions by using `kn func list`. If you want to list functions that have been deployed as Knative services, you can also use `kn service list`.
+
+[NOTE]
+====
+If the namespace is not specified for the following commands, this defaults to the current namespace.
+====
+
+.Procedure
+
+. List existing functions:
++
+[source,terminal]
+----
+$ kn func list -n <namespace> -p <path>
+----
++
+.Example output
+[source,terminal]
+----
+fn-example-io
```

Comment:
Here, the sample output for `kn func list` is a little different. It would be something like this
```
NAME           NAMESPACE  RUNTIME  URL                                                                                      READY
fn-example-io  default    node     http://fn-example-io.default.apps.ci-ln-g9f36hb-d5d6b.origin-ci-int-aws.dev.rhcloud.com  True
```